christian-galarza/lesson-1/car_app/src/main/CarManager.java

```diff
@@ -0,0 +1,59 @@
+package main;
+
+/**
+ * This class build the car.
+ *
+ *  @author Jose Christian Galarza Crespo
+ *  @version 1.0.
+ */
+class CarManager {
+    private Car car= new Car();
+
+    CarManager(double tankCapacity, double performance) {
```

Comment:
It will be easier to have this methods defined in Car class in which case CarManager will not be needed